bin/fkCli.py

```diff
@@ -137,6 +163,46 @@ def process_cmd(client: FaaSKeeperClient, cmd: str, args: List[str]):
     return client.session_status, client.session_id
 
 
+# find the position of the argument in the command (function call like), return -1 if the argument is not a flag
+def kwarg_pos(arg: str, kwargs_array: List[str]):
+    for idx, kwarg in enumerate(kwargs_array):
+        if (type(kwarg) is set) and (arg in kwarg):
+            return idx
+    return -1
+
+# print the command format and flags to user
+def print_cmd_info(cmd: str):
+    args = ""
+    flags = ""
+    for arg in CMD[cmd]:
+        if type(arg) is str:
+            args += f"{arg} "
+        else:
+            flags += f"{arg} "
+    click.echo(f"Command: {args}| Flags: {flags}")
```

Comment:
we can provide users with examples that are syntactically correct. e.g.
`print_cmd_info('create')` generates:
create -s -e /test 0x0